plotting.py

```diff
@@ -0,0 +1,211 @@
+import numpy as np
+import pandas as pd
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+###################################
+
+
+def quick_plot(lidar, points, study_site):
+    """ 
+    Creates a quick plot given a raster image and point file.
+
+    Parameters:
+    ----------
+    lidar: datarray
+        A datarray - Landsat in this case.
+    points: geodataframe
+        A geodatframe containing point locations
+
+    Returns:
+    -------
+    A plot to screen.
+
+    """
+
+    fig, ax = plt.subplots(1, 1, figsize=(10, 6))
+
+    lidar.plot(ax=ax)
+    points.plot(ax=ax, marker='s', markersize=200, color='red')
+
+    # Capitalize study_site for title.
+    title = "{} Study Site:  LIDAR CHM and Study Sites".format(
+        study_site.upper())
+    ax.set_title(title)
+
+    plt.show()
+
+###################################
+
+
+def create_comparison_plots(all_heights_gdf, study_site):
+    """ 
+    Create two plots of lidar vs insitu : MAx and Mean tree heights.
+    Parameters:
+    ----------
+    all_heights_gdf: geodataframe
+        Geodataframe holding merged lidar and insitu stats.
+    study_site: str
+        Name of study_site
+    """
+
+    # Why do I need to convert from a gdf to a df in order to plot?
+    all_heights_gdf = pd.DataFrame(all_heights_gdf)
+
+    fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(8, 4), layout="constrained")
+
+    all_heights_gdf.plot('lidar_max',
+                         'insitu_max',
+                         kind='scatter',
+                         fontsize=14, s=20,
+                         color="purple",
+                         ax=ax1
+                         )
+
+    all_heights_gdf.plot('lidar_mean',
+                         'insitu_mean',
+                         kind='scatter',
+                         fontsize=14, s=20,
+                         color="purple",
+                         ax=ax2
+                         )
+
+    # Auto-detect what value to use for the y & x axis max.
+    # Get the max values for max and mean.
+    lid_mx = all_heights_gdf['lidar_max'].max()
+    lid_mn = all_heights_gdf['lidar_mean'].max()
+    in_mx = all_heights_gdf['insitu_max'].max()
+    in_mn = all_heights_gdf['insitu_mean'].max()
+
```

Comment:
this is a cool component - I have the user manually enter the axis max values, but I like this better